webui/react/src/routes/utils.ts

```diff
@@ -128,3 +128,9 @@ export const linkPath = (aPath: string, external = false): string => {
   }
   return path;
 };
+
+export const paths = {
+  trialDetails: (trialId: number | string, experimentId: number | string): string => {
+    return `/experiments/${experimentId}/trials/${trialId}`;
+  },
+};
```

Comment:
does react router have something like this?

Reply:
I don't think react-router does but this is a great idea to centralize all route generations.

Reply:
we should keep expanding and migrating to this then